spyder/plugins/framesexplorer/widgets/framesbrowser.py

```diff
@@ -48,11 +55,12 @@ def __init__(self, parent, shellwidget, color_scheme):
         self.shellwidget = shellwidget
         self.results_browser = None
         self.color_scheme = color_scheme
-        self.execution_frames = False
-        self.should_clear = False
-        self.post_mortem = False
+        # -1 means never clear, otherwise number of calls
+        self._persistance = -1
```

Comment:
```suggestion
        self._persistence = -1
```

I think you meant `_persistence` instead of `_persistance`. If that's the case, please rename all other references to `_persistance`.